### ISSUE TITLE
Adding background and content scripts in frontend to track words

### ISSUE THREAD

**Issue by Sushi22** [Ayush-projects/chords#42]
@Ayush-projects, I am a participant under GSSOC'21. I would like to complete this part if assigned. :)

**Comment by prykdev** [comment]
Yes you can work on this

**Comment by Sushi22** [comment]
@P-riyanka-prasad Okay :D 
 Can you please assign this issue to me :) i.e. the assign tag

**Comment by prykdev** [comment]
@Ayush-projects Please assign this to @Sushi22

**Comment by Ayush-projects** [comment]
Yeah, Assigned. Try to do it in pure javascript instead of Jquery.

**Comment by Sushi22** [comment]
> Yeah, Assigned. Try to do it in pure javascript instead of Jquery.

Sure.

**Comment by Sushi22** [comment]
@Ayush-projects I have made the Pull Request . Please review :)
